apps/ai-image-generator/backend/src/transform-images.ts

```diff
@@ -0,0 +1,158 @@
+import * as nodeFetch from 'node-fetch';
+import { default as sharp } from 'sharp';
+import { areEqualColors, toRGBA, toSharp } from './utils';
+
+interface TransformedImages {
+  image: Buffer;
+  mask: Buffer;
+}
+
+export const ERASE_COLOR: sharp.RGBA = { r: 174, g: 193, b: 204 };
```

Comment:
@lilbitner Can you confirm this is the exact values we want

Reply:
yeah so at this very moment in master this is the correct color (gray400)! I think we might adjust this to `gray300` with some styling updates on the frontend, but makes sense to me to keep this aligned with master right now to test functionality.

Reply:
👍 Yep, and we can evolve this pretty easily as time goes on. (We coud even make it a param to the app action if we wanted to further customability)